src/mousev1_to_nwb/abdeladim_2023/abdeladim_2023_convert_session.py

```diff
@@ -1,71 +1,78 @@
 """Primary script to run to convert an entire session for of data using the NWBConverter."""
 from pathlib import Path
 from typing import Union
+import glob
 import datetime
 from zoneinfo import ZoneInfo
 
 from neuroconv.utils import load_dict_from_file, dict_deep_update
 
-from mousev1_to_nwb.abdeladim_2023 import Abdeladim2023NWBConverter
-
-
-def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str, Path], stub_test: bool = False):
+from abdeladim_2023imaginginterface import Abdeladim2023SinglePlaneImagingInterface
+from abdeladim_2023nwbconverter import Abdeladim2023NWBConverter
+from abdeladim_2023nwbconverter import get_default_segmentation_to_imaging_name_mapping
 
+def session_to_nwb(
+    data_dir_path: Union[str, Path], output_dir_path: Union[str, Path], session_id: str, stub_test: bool = False
+):
     data_dir_path = Path(data_dir_path)
     output_dir_path = Path(output_dir_path)
     if stub_test:
         output_dir_path = output_dir_path / "nwb_stub"
     output_dir_path.mkdir(parents=True, exist_ok=True)
-
-    session_id = "subject_identifier_usually"
+    subject_id = "unknown"  # TODO ask for subject_id
```

Comment:
you could add this as a function argument to `session_to_nwb()` if the source_data doesn't include this info